Eat_and_Greet/src/screens/ChatsPage/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { View } from 'react-native';
+import ChatCard from '../../comps/ChatCard/ChatCard'; 
+
+import { Toolbar } from '../../comps/Toolbar/toolbar';
+
+import styles from '../../style';
+
+function HomeScreen() {
+
+    return (
+      <View style={styles.ScreenContainer}>
+      <View style={styles.OuterBox}>
+          
+        <ChatCard senderName="@johndoe" message="Hello there!" timestamp="10:30 PM"/>
```

Comment:
I think these should be mapped out from a separate array like the screens on App.tsx. Not necessary now, but will help us implement the functionality later when we need it to be mapped out.

Reply:
will make a ticket for that

Reply:
perfect thanks